pages/featurelist_test.py

```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
 
 
+from framework import ramcache
```

Comment:
Can this be put down on line 23 with the other imports for our app code?
You are proving that the comment on the next line is not strictly true.

Reply:
Done